Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -220,7 +224,7 @@
           {
             "type": 1,
             "content": {
-              "json": "{Message}\r\n_Azure Cosmos DB accounts_\r\n<br />\r\n<br />"
+              "json": "{Message}\r\n_Azure Cosmos DB Accounts_\r\n<br />\r\n<br />"
```

Comment:
just curious, does 'Accounts' have to be capitalized? Might be marketing thing that we left accounts lowercase on purpose